### PR TITLE
User can chose grid layout properties.

### PR DESCRIPTION
**Pull request by MariaAdner**
User can choose from a radio button menu wether he want to have a grid for x and y axis. User can also select the line style for the grid.

### CODE REVIEW

**Review by MareikeFla — COMMENTED** [review]
Hi, I left a few comments, but in addition to that I have a question and a problem. 
The section with "Show grid (helper lines) for axes" is visible from the beginning and that is kind of confusing because the other steps in your app are only visible after finishing the previous step, is that intended? 
And I can not see any changes in the grid layout when I select something from the drop down menus - radio buttons and drop down menu are working as intended (i guess) but I can not see any changes?

File: Components/Container/index.js [hunk 1]
```diff
@@ -32,4 +32,13 @@ export const Container = styled.section`
...
     `};
+
+  ${(props) =>
+    props.$direction === "row" &&
+    css`
+      display: flex;
+      flex-direction: row;
+      align-items: flex-start;
+      gap: 1rem;
+    `};
```
[the remark this answers is not in the document]
Comment (MariaAdner) [reply Components/Container/index.js:43]: That's a good point! We will consider it as our project grows and refactor it, when we have all different styling option ready! Then it is easier to decide, in which way we can summarize things.

File: Components/GridProperties/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { Container } from "../Container";
+import DropDownMenu from "../DropDownMenu";
+import { lineStyleArray } from "../../lib/listOfLineProperties";
+
+export default function GridProperties({ grid, onGridChange }) {
+  return (
+    <>
+      <Paragraph>Show grid (helper lines) for axes: </Paragraph>
+      <Container $wrap="wrap">
+        <Container $centered="center">
```
[the remark this answers is not in the document]
Comment (MariaAdner) [reply Components/GridProperties/index.js:12]: It looks like doubled, however it is needed.

File: Components/InputTypeRadio/index.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  return (
+    <>
+      <RadioButton>
+        <input
+          type="radio"
+          name={nameString}
+          value={valueString}
+          id={idString}
+          onChange={onChange}
+          checked={checked()}
```
[the remark this answers is not in the document]
Comment (MariaAdner) [reply Components/InputTypeRadio/index.js:23]: See reply above.

**Comment by MariaAdner** [comment]
> Hi, I left a few comments, but in addition to that I have a question and a problem. The section with "Show grid (helper lines) for axes" is visible from the beginning and that is kind of confusing because the other steps in your app are only visible after finishing the previous step, is that intended? And I can not see any changes in the grid layout when I select something from the drop down menus - radio buttons and drop down menu are working as intended (i guess) but I can not see any changes?


Thank you for your review! From our previous US you are used to see something in the Plot in the end, however, this time, it is not part of the US - therefore the component is just displayed at the UI -> Later we will change that. Together with other components all steps will be displayed in sequence and the Plot gets updated according to the settings of the user.

**Pull request merged**